Tests/XcodeGenKitTests/ProjectSpecTests.swift

```diff
@@ -74,6 +74,17 @@ class ProjectSpecTests: XCTestCase {
                 configSettings: ["invalidConfig": [:]],
                 groups: ["invalidSettingGroup"]
             )
+            
+            $0.it("fails with invalid XcodeGen version") {
+                let minimumVersion = try! Version("1.11.1")
```

Comment:
No biggie, but these tests are allowed to throw so no need for the exclamation marks

Reply:
Ahh, true. I'll adjust it.